features/frameworks/TARGET_PSA/val.h

```diff
@@ -169,9 +169,9 @@
 
 /* enums */
 typedef enum {
-    NONSECURE = 0x0,
-    SECURE    = 0x1,
-} security_t;
+    CALLER_NONSECURE = 0x0,
+    CALLER_SECURE    = 0x1,
+} caller_security_t;
```

Comment:
Is this changing actual enumerated types used by a developer ? Or is val.h a test only header file ?

Reply:
`val.h` is a test only header file.